LABS.md

```diff
@@ -20,7 +20,7 @@ download: git-scm.com, or use your favourite package manager
 2. start tracking the file using `git add <filename>`
 3. see the blob appear in gitviz and/or the `.git/objects/` subfolder
 4. check the status using `git status`
-5. commit the staged files using `git commit -m 'your message'`
+5. commit the staged files using `git commit -m "your message"`
```

Comment:
what is this change for? whouldn't it be a better idea to make one generic command on double quotes to be used throughout the document in CMD?

Reply:
I have checked, this is the only git command having single quotes